ci/process_notebooks.py

```diff
@@ -157,6 +163,18 @@ def has_solution(cell):
     )
 
 
+def has_colab_badge(cell):
+    """Return True if cell has a Colab badge as an HTML element."""
+    return "colab_badge.svg" in cell["source"]
+
+
+def redirect_colab_badge(cell):
+    """Modify the Colab badge to point at the master branch on Github."""
+    cell_text = cell["source"]
+    p = re.compile(r"^(.+/NeuromatchAcademy/course-content/blob/)\w+(/.+$)")
```

Comment:
I think this wouldn't work if someone pulls in from a fork.

Reply:
That's true. But is that the workflow we're going to use? I'm not sure this bit of processing can solve all problems — we'd also have to replace the repository name in the badge then too.

Canonical Colab links should probably be hand-curated in the README / on the website.